apps/site/pages/docs/api-extensions/troubleshooting.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Troubleshooting'
+sidebar_label: 'API Extensions'
+---
+
+import { Callout, Step, Steps } from 'nextra-theme-docs'
+import { ExtendableQueryTable } from './../../../components/ExtendableQueryTable'
+
+<header>
+
+# Troubleshooting
+
+</header>
+
+## GraphQL changes not visible during development
+If your GraphQL changes aren't appearing during development, this is because the changes you have made since you started the development server (`yarn dev`) are probably not optimized yet. 
+Follow these steps to trigger the optimization:
+
+- Run `yarn generate` (recommended) or `yarn run faststore generate-graphql`.
```

Comment:
I'm not sure if we have the `yarn generate`command exposed in the starter anymore. @icazevedo can confirm that?

Reply:
I think we should update it to have it, yes!